### PR TITLE
BaSP-TG-29: Projects Unit Test get all, get id

### PR DESCRIPTION
**Pull request by LucianoFranchino**

### CODE REVIEW

File: src/seeds/projects.js [hunk 1]
```diff
@@ -1,133 +1,51 @@
...
+  }],
+}, {
+  _id: mongoose.Types.ObjectId('6350160d76dd03ff29e75dd3'),
+  name: 'Nan',
+  description: 'quisque porta',
+  startDate: '2012-01-09T00:00:00.000+00:00',
+  endDate: '2022-12-18T00:00:00.000+00:00',
+  clientName: 'Gabtype',
+  employees: [{
+    _id: mongoose.Types.ObjectId('634d7e5757bc44f21a345462'),
```
Comment (paurinaldi) [inline src/seeds/projects.js:47]: shouldn't we be testing employeeId?

**Review by paurinaldi — CHANGES_REQUESTED** [review]

File: src/tests/projects.test.js [hunk 2]
```diff
@@ -1 +1,42 @@
...
+import request from 'supertest';
+import app from '../app';
+import Projects from '../models/Projects';
+import projectsSeed from '../seeds/projects';
+
+const idProject = '635015885581eb421df09402';
+const idInvalid = '635015885581eb421df09404';
+
+describe('Projects - Unit test', () => {
+});
```
Comment (paurinaldi) [inline src/tests/projects.test.js:10]: This should be closed at the bottom of all other describes. After line 42

Comment (LucianoFranchino) [reply src/tests/projects.test.js:10]: Resolved

File: src/app.js [hunk 3]
```diff
@@ -4,7 +4,7 @@ import routes from './routes/index';
 const app = express();
 
 app.use(express.json());
-app.use('/api', routes);
+app.use('/', routes);
```
[the remark this answers is not in the document]
Comment (LucianoFranchino) [reply src/app.js:7]: Resolved

File: src/seeds/projects.js (hunk 1 above)
Comment (LucianoFranchino) [reply src/seeds/projects.js:47]: I had understood that within the project you had to check the id, role and rate of the employee

Comment (LucianoFranchino) [reply src/seeds/projects.js:47]: Resolved

**Review by paurinaldi — APPROVED** [review]

**Pull request merged**